app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class VotesController < ApplicationController
+  def create
+    @post = Post.find(params[:votable_id])
+    service = CreateVoteService.new(get_votable,current_user,params)
```

Comment:
general style guideline: separate method parameters with spaces after the deliminating commas